lib/semantics/expression.h

```diff
@@ -71,15 +74,28 @@ class ExpressionAnalysisContext {
 
   semantics::SemanticsContext &context() const { return context_; }
 
+  FoldingContext &GetFoldingContext() const {
+    return context_.foldingContext();
```

Comment:
I think there is a place in resolve-names.cc that can use FoldingContext() too.

Reply:
There's a couple, will do.